### PR TITLE
Updates faulty link (#1422) 

### PR DESCRIPTION
**Pull request by kevinnerio**
https://rawgit.com/kevinnerio/kevinnerio.github.io/master/#!index.md

### CODE REVIEW

File: pages/vi/vi-first-steps.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ If you are selected for the internship after completing the steps, you will be o
 
 Once accepted, you and your team will work on an assignment, and we’ll switch up the assignments each week. As part of this internship, you will be working with software and languages including Git, GitHub, Gitter, Markdown, Vagrant, VirtualBox, Command Line/Terminal, Command Line/Terminal Scripts, Vim, CouchDB, Docker, HTML5, Javascript, and Node.js.
 
-**NOTE**: This is an unpaid but intensive internship that requires 16 hours of work each week. More information about the internship can be found [here](http://open-learning-exchange.github.io/#!pages/vi/vi-faq.md). If you have further questions, don’t hesitate to ask other members in the Gitter [chat](vi-chat.md)!
+**NOTE**: This is an unpaid but intensive internship that requires 16 hours of work each week. More information about the internship can be found [here](https://www.indeed.com/cmp/Open-Learning-Exchange,-Inc./jobs). If you have further questions, don’t hesitate to ask other members in the Gitter [chat](vi-chat.md)!
```
Comment (yaoxing1990) [inline pages/vi/vi-first-steps.md]: Good job!

**Comment by yaoxing1990** [comment]
@kevinnerio You should start working from another branch but master. And there are too many commits with only a few file changes, some of which are confusing.

**Comment by kevinnerio** [comment]
@yaoxing1990  Hey, I know there is many commits that was due to me trying to figure out some things. I am working on getting familiar with github in order to minimize that error.

**Comment by kevinnerio** [comment]
@yaoxing1990 Hey I dont know if this PR hasnt been closed but it seems to be limiting my ability to make a new PR.

**Comment by lorugant** [comment]
@kevinnerio It looks like you are working from master branch. Create a new branch, make the changes and then create a pull request. Please don't make changes to the master branch.

**Comment by kevinnerio** [comment]
@lorugant Hey sorry for the late response. I am back from the holidays. I need some clarification with branches, do I make the extra branch on terminal on my local branch or on my github profile? Which is the preferred method?

**Comment by kylemathias** [comment]
@kevinnerio From your forked repository, you create a new branch while you are on the master branch.

**Comment by xyb994** [comment]
@kevinnerio 
- Since you are just starting out and your master branch is not "clean" anymore, the easiest way is to delete kevinnerio.github.io repository and fork again
- Creating branch in command line is preferred, we were branching using GitHub's interface on Step 3 just so we could get familiar with GitHub. When we work with other projects in the future, it won't be as simple as markdown in First Steps which we could preview if changes work right away

**Comment by kevinnerio** [comment]
@xyb994 I am not.

**Pull request closed without merging**